rollingpin/args.py

```diff
@@ -81,6 +81,15 @@ def _add_iteration_arguments(config, parser):
         dest="pause_after",
     )
 
+    iteration_group.add_argument(
+        "--timeout",
+        default=None,
```

Comment:
Yeah that's what it is.  The other code checks if this is `None` and uses the ini file if so.